docs/guide/extending_online_editor.md

```diff
@@ -255,6 +255,85 @@ In the example above, the template files for the front end could be:
 
 Templates for Content View in the Back Office would be `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_word.html.twig` and `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_block.html.twig` respectively (assuming Admin SiteAccess uses the `admin` theme).
 
+## Custom attributes and classes
+
+You can add custom attributes and CSS classes to elements in the Online Editor.
+
+The available elements are:
+
+- `embedinline`
+- `embed`
+- `formatted`
+- `heading`
+- `embedimage`
+- `ul`
+- `ol`
+- `li`
+- `paragraph`
+- `table`
+- `tr`
+- `td`
+
+### Custom attributes
+
+Custom attributes are configured under the `fieldtypes.ezrichtext.attributes` key.
+
+A custom attribute can belong to one of four types: `choice`, `boolean`, `string`, or `number`.
+You can also set each attribute to be `required` and set its `default_value`.
+
+For the `choice` type, you must provide an array of available `choices`.
+`multiple` enables you to choose whether more than one option can be selected.
```

Comment:
```suggestion
Adding `multiple` enables you to choose whether more than one option can be selected.
```
Just to start a sentence with a capital letter, nothing more than that.

**Question:** Is `multiple` mandatory for `choices`? Meaning, the dev has to provide either `multiple: false` or `multiple: true`?

Reply:
> **Question:** Is `multiple` mandatory for `choices`? Meaning, the dev has to provide either `multiple: false` or `multiple: true`?

It's not required - by default it's true.